python/ray/dashboard/client/src/pages/dashboard/memory/Memory.tsx

```diff
@@ -64,12 +89,18 @@ const makeUngroupedEntries = (
     orderBy === null
       ? allEntries
       : stableSort(allEntries, getComparator(order, orderBy));
-  return sortedEntries.map((memoryTableEntry, index) => (
-    <MemoryTableRow
-      memoryTableEntry={memoryTableEntry}
-      key={`mem-row-${index}`}
-    />
-  ));
+  return (
+    <React.Fragment>
+      {" "}
+      {sortedEntries.slice(0, visibleEntries).map((memoryTableEntry, index) => (
+        <MemoryTableRow
+          memoryTableEntry={memoryTableEntry}
+          key={`mem-row-${index}`}
```

Comment:
Why not just index?

Reply:
Good question.
https://reactjs.org/docs/lists-and-keys.html stable keys are recommended, but I'm realizing the entry I create isn't stable anyway, so it may as well just be the index.